README.md

```diff
@@ -115,9 +115,16 @@ Or with the `WEB_CONCURRENCY` environment variable:
 $ WEB_CONCURRENCY=3 puma -t 8:32
 ```
 
+When using a config file, most applications can simply set `workers :auto` (requires the `concurrent-ruby` gem) to match the number of worker processes to the available processors:
+
+```ruby
+# config/puma.rb
+workers :auto
+```
+
 Note that threads are still used in cluster mode, and the `-t` thread flag setting is per worker, so `-w 2 -t 16:16` will spawn 32 threads in total, with 16 in each worker process.
```

Comment:
The positioning of this line now seems awkward. It might read better before 'When using a config file ...'?